system-int-test/test/cmr/system_int_test/search/variable/variable_revisions_search_test.clj

```diff
@@ -131,20 +131,36 @@
        [var1-2-tombstone var1-3 var2-1 var2-2 var3]
        (search/find-refs :variable {:all-revisions true}))
 
-      (testing "force delete cascade to variable association"
-        (variables/associate-by-concept-ids token
-                                            (:concept-id var1-3)
-                                            [{:concept-id (:concept-id coll1)}])
-        (index/wait-until-indexed)
-        ;; search collections by variable native-id found the collection
-        (d/refs-match? [coll1] (search/find-refs :collection {:variable_native_id "var1"}))
-
-        ;; force delete the latest revision of the variable cascade to variable association
-        (metadata-db/force-delete-concept (:concept-id var1-3) (:revision-id var1-3))
-        (index/wait-until-indexed)
-        
-        ;; search collections by variable native-id no longer found the collection
-        (d/refs-match? [] (search/find-refs :collection {:variable_native_id "var1"}))))))
+      ;; Associate a collection and variable
+      (variables/associate-by-concept-ids token
+                                          (:concept-id var1-3)
+                                          [{:concept-id (:concept-id coll1)}])
+
```

Comment:
Someone likes their newlines in Clojure code ;-)

Reply:
https://github.com/bbatsov/clojure-style-guide#no-blank-lines-within-def-forms

Reply:
Don't tell anyone I removed them all.